README.adoc

```diff
@@ -0,0 +1,42 @@
+[[git-plugin]]
+= Tool Environment Plugin
+:toc: macro
+:toc-title:
+
+link:https://ci.jenkins.io/job/Plugins/job/toolenv-plugin/job/master/[image:https://ci.jenkins.io/job/Plugins/job/toolenv-plugin/job/master/badge/icon[Build Status]]
+link:https://github.com/jenkinsci/toolenv-plugin/graphs/contributors[image:https://img.shields.io/github/contributors/jenkinsci/toolenv-plugin.svg?color=blue[Contributors]]
+link:https://github.com/jenkinsci/toolenv-plugin/releases/latest[image:https://img.shields.io/github/release/jenkinsci/toolenv-plugin.svg?label=changelog[GitHub release]]
+link:https://plugins.jenkins.io/toolenv[image:https://img.shields.io/jenkins/plugin/i/toolenv.svg?color=blue&label=installations[Jenkins Plugin Installs]]
+
+[#introduction]
```

Comment:
asciidoc will automatically create an identifier for heading with the same text as the heading
```suggestion
```